stdlib/experimental/oee/oee.flux

```diff
@@ -1,17 +1,94 @@
+// Package oee provides functions for calculating overall equipment effectiveness (OEE). 
+// 
+// introduced: 0.112.0
+// 
 package oee
 
 
 import "contrib/tomhollingworth/events"
 import "experimental"
 
-// computeAPQ computes availability, performance, quality and overall equipment effectiveness (oee).
-// productionEvents - a stream of start/stop events for the production process. Each row contains
-//   a _time and state that indicates start and stop events.
-// partEvents - a stream of part counts. Each row contains cumulative counts where column partCount
-//   represents total number of produced parts and badCount number of parts that did not meet quality standards.
-// runningState - production event or state value that corresponds to equipment running state
-// plannedTime - total time that equipment is expected to produce
-// idealCycleTime - theoretical minimum time to produce one part
+// computeAPQ computes availability, performance, and quality (APQ)
+// and overall equipment effectiveness (OEE) using two separate input streams—**production events**
```

Comment:
I guess we can't use an em-dash in our docs, I wish we could.

Reply:
Replace with a colon.